app/models/course/assessment/question.rb

```diff
@@ -9,6 +9,8 @@ class Course::Assessment::Question < ActiveRecord::Base
   has_many :answers, class_name: Course::Assessment::Answer.name, dependent: :destroy,
                      inverse_of: :question
   has_and_belongs_to_many :skills
+  has_many :submission_questions, class_name: Course::Assessment::SubmissionQuestion.name,
+                                  inverse_of: :question
```

Comment:
This could potentially wipe out a lot of comments, but I suppose it's no worse than having `dependent: :destroy` on answers.